docs/pages/access-controls/reference.mdx

```diff
@@ -357,9 +357,39 @@ allow:
     - resources:
         - token
       verbs: [list, create, read, update, delete]
+```
+
+### Allowing access to token resources
 
+You should note that if you configure a role that allows tokens to be created, a user assigned to the 
+role can create any type of token. 
+For example, you might create a role with the following configuration to enable assigned
+users to enroll servers:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: enroll-servers
+spec:
+  allow:
+    node_labels:
+      'env': 'us-lab'
+    rules:
+      - resources: [token]
+        verbs: [list, create, read, update, delete]
+  deny: {}
 ```
 
+With these permissions, users assigned this role can generate any type of token, including a token to enroll 
```

Comment:
I wouldn't say "any type of token" because technically it's all the same type of token (provision token). What we mean to say is that the token can include any system roles (ssh, db, windowsdesktop, etc) and that it's not possible to create a role that restricts token creation to a subset of roles.